stdlib/public/SDK/Network/NWPath.swift

```diff
@@ -75,6 +75,7 @@ public struct NWInterface : Hashable, CustomDebugStringConvertible {
 		}
 	}
 
+	/// The interface type wifi, cellular, wiredEthernet, and loopback.
```

Comment:
Great. Last thing: to clarify my previous comment again, please remove "wifi, cellular, wiredEthernet, and loopback". That information is inherent in the fact that InterfaceType is an enumeration with only those cases.